exercises/pangram/description.md

```diff
@@ -2,5 +2,5 @@ Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan
 "every letter") is a sentence using every letter of the alphabet at least once.
 The best known English pangram is "The quick brown fox jumps over the lazy dog."
 
-The alphabet used is ASCII, and case insensitive, from 'a' to 'z'
-inclusively.
+The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
+insensitive. Input should not contain non-ASCII symbols.
```

Comment:
Good call, I couldn't place my finger on this earlier but you captured my niggling feeling well!